platform/android/java/lib/src/org/godotengine/godot/input/GodotEditText.java

```diff
@@ -165,6 +168,10 @@ private void handleMessage(final Message msg) {
 					}
 					edit.setInputType(inputType);
 
+					if (acceptCharacters != null) {
+						edit.setKeyListener(DigitsKeyListener.getInstance(acceptCharacters));
```

Comment:
Can you use [`DigitsKeyListener.getInstance(Locale)`](https://developer.android.com/reference/android/text/method/DigitsKeyListener#getInstance(java.util.Locale)) instead.

Reply:
- API level 26 (Android 8.0)
- does not work on Samsung OS (video below) and Locale = `de_DE` (Germany)
- but I think it would be possible with [Gboard App](https://play.google.com/store/apps/details?id=com.google.android.inputmethod.latin), should I still use Locale?

```java
	if (acceptCharacters != null) {
		Log.d("Locale", "getDefault: " + Locale.getDefault());
		if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.O) {
			edit.setKeyListener(DigitsKeyListener.getInstance(Locale.getDefault()));
		}
		else {
			edit.setKeyListener(DigitsKeyListener.getInstance(acceptCharacters));
		}
	}

```

Android 13, Samsung Tab S7


https://github.com/godotengine/godot/assets/41921395/c80cb1d3-4823-4132-9346-57dba65d9376

Reply:
Yeah, this is an issue with the Samsung keyboard not with the api itself.

The issue with specifying the list ourselves is that we may end be omitting a character that's used by a locale not familiar to us.
For example, the empty space is accepted in some locales

https://docs.oracle.com/cd/E19455-01/806-0169/overview-9/index.html#:~:text=Great%20Britain%20and%20the%20United,also%20called%20the%20radix%20character.